LogisticApp/ViewModel/Forms/PersonFormViewModel.cs

```diff
@@ -68,8 +68,42 @@ public override void loadData(BaseEntity entity)
         //TODO add validation if there will be enough time
         public override void save()
         {
-            _addressViewModel.save();
-            Creator.createOrUpdate("person", this.serializeData());
+            if (canSave())
+            {
+                _addressViewModel.save();
+                Creator.createOrUpdate("person", this.serializeData());
+            }
+            else
+                Console.WriteLine("Empty textbok or incorrect data.");
```

Comment:
using WriteLine in window app is not the best idea 😄

Reply:
yup, wasn't sure how to use this method in context of a program and wanted you to check this out :)